clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystemCore.java

```diff
@@ -34,91 +37,85 @@
 interface TachyonFileSystemCore {
 
   /**
-   * Creates a file with the provided block size as the standard block size of the file. If the
-   * file's parent directories do not exist, they will be created if the recursive flag is set.
+   * Creates a file.
    *
    * @param path the path of the file to create in Tachyon space
-   * @param blockSize the block size in bytes, must be greater than 0
-   * @param recursive whether or not to create parent directories if required
-   * @param ttl time to live for file expiration
    * @return the file id that identifies the newly created file
-   * @throws BlockInfoException if the block size is less than or equal to 0
-   * @throws FileAlreadyExistException if the path already exists as a file in Tachyon
-   * @throws InvalidPathException if the path is not a valid Tachyon path
-   * @throws IOException if the master is unable to create the file
+   * @throws IOException if a non-Tachyon exception occurs
+   * @throws TachyonException if a Tachyon exception occurs
    */
-  long create(TachyonURI path, long blockSize, boolean recursive, long ttl) throws
-      BlockInfoException, FileAlreadyExistException, InvalidPathException, IOException;
+  long create(TachyonURI path) throws IOException, TachyonException;
```

Comment:
I think the idea was to make the core interface only contain the fully customized methods (non-default, with options). What was the reason to add the default methods here? (applies to all default methods).